packages/cli/src/main.ts

```diff
@@ -1,8 +1,100 @@
 #!/usr/bin/env node
-import { Command } from 'commander';
+import { Command, createOption, Option } from 'commander';
 import { node } from './commands/node';
+import { TaskParam, TaskArguments, initialKuai, paramTypes, KuaiArguments } from '@kuai/core';
 
-const program = new Command();
+const KUAI_GLOBAL_PARAMS: Array<TaskParam> = [
+  {
+    name: 'config',
+    description: 'A Kuai config file.',
+    type: paramTypes.path,
+    isFlag: false,
+    isOptional: true,
+  },
+];
 
-program.addCommand(node);
-program.parse();
+function parseTaskParams(param: TaskParam<TaskArguments>): Option {
+  const type = (() => {
+    if (param.type.name === 'boolean') {
+      return '';
+    }
+
+    return ` <${param.type.name}>`;
+  })();
+
+  return createOption(`--${param.name}${type}`, param.description);
+}
+
+const main = async () => {
+  const program = new Command();
+
+  KUAI_GLOBAL_PARAMS.map((param) => parseTaskParams(param)).forEach((option) => program.addOption(option));
+
+  program.addCommand(node);
+
+  const args: KuaiArguments = (() => {
+    const result: KuaiArguments = {};
+
+    const configPathFlagIndex = process.argv.slice(0, -1).lastIndexOf('--config');
+
+    if (configPathFlagIndex !== -1) {
+      Object.assign(result, { configPath: process.argv[configPathFlagIndex + 1] });
```

Comment:
The config path may be set to null, is it affecting to run tasks?

Reply:
No effect